app/logger.py

```
@@ -9,6 +12,28 @@
 from loguru import logger as log
 import uvicorn.logging
 
+# Import configuration (contains sentry toggles)
+from app.config import get_config
+
+try:
+    import sentry_sdk
+    from sentry_sdk.integrations.logging import LoggingIntegration
+except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)